app/helpers/publish_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 module PublishHelper
   def markdown(source)
+    return '' if source.blank?
```

Comment:
If we remove this line these specs fail:

```
rspec ./spec/features/publish/courses/editing_degree_requirements_spec.rb:10 # Editing degree requirements requires minimum degree classification
rspec ./spec/features/publish/courses/editing_degree_requirements_spec.rb:25 # Editing degree requirements does not require minimum degree classification
rspec ./spec/features/publish/courses/editing_degree_requirements_spec.rb:37 # Editing degree requirements primary course requires minimum degree classification
rspec ./spec/features/publish/courses/editing_degree_requirements_spec.rb:47 # Editing degree requirements primary course does not require minimum degree classification
rspec ./spec/features/publish/courses/gcse_equivalency_spec.rb:6 # GCSE equivalency requirements a provider completes the gcse equivalency requirements section
rspec ./spec/features/publish/courses/gcse_equivalency_spec.rb:28 # GCSE equivalency requirements a provider views course pages with course 2 GCSE requirements
rspec ./spec/features/publish/courses/gcse_equivalency_spec.rb:34 # GCSE equivalency requirements a provider views course pages with course 3 GCSE requirements
```

Just adding because I'll probably (or someone else) will see this line was added and wonder which spec to identify that could fail.